### ISSUE TITLE
Onboarding: The signup and create site flow does not land the user on the My Site screen with Quick Start

### ISSUE THREAD

**Issue by antonis** [wordpress-mobile/WordPress-Android#15777]
### Expected behavior
When the user finishes the signup flow and creates a new site they get prompted for a quick walkthrough (*Quick Start*) and land on the *My Site* screen.

### Actual behavior
When the user finishes the signup flow and creates a new site they land on the *Notifications* screen with the *Welcome to WordPress.com!* notification. This notification is not that useful on the mobile and the included links are not actionable.

Note: The *Quick Start* is available when the user logs in again.

<details>
<summary>Welcome notification screenshot</summary>

![device-2021-12-28-125647](https://user-images.githubusercontent.com/304044/147559353-ab60c2c1-5e94-45b2-a78f-fd7fec5f5d93.png)
</details>

### Steps to reproduce the behavior
1. On a new app install sign up and select to create a site
2. Follow the steps to create the site
3. *Notice* that after site creation you land on the *Notifications* screen

[Video recording](https://cldup.com/HbBV-5lVNe.mp4)

*Note: On iOS the user lands on quick start / my site screen ([video](https://cloudup.com/crz40hMyHXJ)) as expected*

**Comment by ovitrif** [comment]
@antonis I've tested the WordPress Android app and tried to reproduce the issue with the following setups:

### 1. Local build ➡️ commit [a7316ed](https://github.com/ovitrif/WordPress-Android/commit/a7316eddae34a5ac1525255a471f5d974a5fde1f) from `trunk`
- Pixel XL Emulator on Android API level 32: Couldn't reproduce (tested 3 times)
- Samsung Galaxy S8+ Android AP levelI 28 : Couldn't reproduce (tested 2 times)

### 2. Official App downloaded from the Play Store ➡️ Version 19.0 (1157)
- Samsung Galaxy S8+ Android API level 28: Couldn't reproduce (tested 3 times)

I'm going to further investigate this tomorrow 🧐

Here's a preview on how the flow went on the latest version of the WordPress Android App:

https://user-images.githubusercontent.com/4588074/151860810-2f700ba3-f5d4-4e30-ba69-ecfcc44dd544.mov

**Comment by antonis** [comment]
Hey @ovitrif 👋 
Thank you for investigating this 🙇 

> @antonis I've tested the WordPress Android app and tried to reproduce the issue with the following setups:

I tried with the latest version of the App and I'm not able to reproduce the issue either. I also tried with [18.9](https://github.com/wordpress-mobile/WordPress-Android/releases/tag/18.9) that I originally tested (and recorded above) and wasn't able to reproduce the issue either. Probably it's a configuration change that affected this.
I'll close the issue on the Android side.
